### PR TITLE
Use get_partition_indices() in case weights example

### PR DESCRIPTION
**Pull request by kelly-sovacool**
We never showed how to use `get_partition_indices()` in the intro vignette after adding it in https://github.com/SchlossLab/mikropml/pull/200.

## Checklist

(~Strikethrough~ any points that are not applicable.)

- ~[ ] Write unit tests for any new functionality or bug fixes.~
- ~[ ] Update docs if there are any API changes:~
  - ~[ ] roxygen comments~
  - [x] vignettes
- ~[ ] Update `NEWS.md` if this includes any user-facing changes.~ 
- [x] The check workflow succeeds on your most recent commit. **This is always required before the PR can be merged.**

### CODE REVIEW

File: vignettes/introduction.Rmd [hunk 1]
```diff
@@ -241,11 +241,12 @@ we think the defaults are a good starting point [@topcuoglu_framework_2020].
...
-`training_frac`. However, in some cases you might wish to control exactly which
-observations are in the training set. You can instead assign `training_frac` a
-vector of indices that correspond to which rows of the dataset should go in the
-training set (all remaining sequences will go in the testing set).
-Here's an example with ~80% of the data in the training set:
+`training_frac` using `get_partition_indices()`. However, in some cases you
+might wish to control exactly which observations are in the training set. You
+can instead assign `training_frac` a vector of indices that correspond to which
+rows of the dataset should go in the training set (all remaining sequences will
+go in the testing set). Here's an example with ~80% of the data in the training
```
Comment (courtneyarmour) [inline vignettes/introduction.Rmd:248]: here it says the example has ~80% of data in training set but below in the code chunk training frac is set to 0.7?

Comment (kelly-sovacool) [reply vignettes/introduction.Rmd:248]: @courtneyarmour I think you're looking at the code chunk at L417, but the chunk this refers to at L252 does use 0.8.

Comment (courtneyarmour) [reply vignettes/introduction.Rmd:248]: ah right missed that there was a chunk of code hidden!

**Review by courtneyarmour — APPROVED** [review]

**Comment by kelly-sovacool** [comment]
I'm working on resolving these merge conflicts locally, then will merge it. Thanks @courtneyarmour!

**Pull request merged**